src/main/java/org/hisp/dhis/antlr/function/AntlrFunctionLog.java

```diff
@@ -0,0 +1,57 @@
+package org.hisp.dhis.antlr.function;
```

Comment:
In the parser code so far I've used the terms "operator" and "function" to mean the same two things as in describing expressions in a programming language. So for example in Expression.g4 there is a section for "operators", which are symbols with arguments before and/or after, and then "functions", which are a function name followed by parentheses that may contain arguments. In all the parser action routines so far, I've put operators and functions in different subdirectories. This is perhaps a bit arbitrary, but is one way of organizing the code. I was staying with this way of organizing.

Alternately, we could move the parser action routines for log() and log10() into dhis2-core. But since they were simple mathematical functions, I thought we could put them here instead -- just in case program rules ever wanted to use them.